dockerbot.py

```diff
@@ -23,16 +45,85 @@ def handle(msg):
 
     logger.info(f"[{message_id}] Got command: {command}")
 
-    if command == '/sign':
+    if command == '/commands':
```

Comment:
change the commands to /? (much more user friendly) 
```suggestion
    if command == '/?':
```